### DISCUSSION TITLE
Seeking General Method for Updating Transporter Concentration from 9.1 to 10

### DISCUSSION THREAD

**Discussion by prvmalik** [Open-Systems-Pharmacology/Forum#1083] [Questions & Problems]
Hi,

Seeking to learn what would be the best way to correct transporter reference concentrations after the v10 update and if it is possible to maintain model integrity from v9.1 given the structural changes. Thanks

Paul

**Comment by StephanSchaller** [comment]
@PavelBal @sfrechen, maybe we can upload some "guidance"?

**Comment by Yuri05** [comment]
Instructions provided by @sfrechen (s. https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library/issues/62#issuecomment-1014460184)
